qa/tasks/cephfs/test_cephfs_shell.py

```diff
@@ -1024,7 +1063,7 @@ def test_setting_opt_after_reading_conf(self):
         # now: vim
         # editor: vim
         final_editor_val = self.get_cephfs_shell_cmd_output(
-            cmd='set editor %s, set editor' % (self.editor_val),
+            cmd='set editor %s, set editor' % self.editor_val,
```

Comment:
f-string method can be used here.

Reply:
It would be more like code enhancement rather than pep8 fixes. There would be many such instances where f-string might be an option but this PR doesn't address that topic. Using non-fstrings is not pep8 violation IMO.

Reply:
It was just a suggestion to maintain uniformity w.r.t this file and as I can see there are very few such instances.Rest your choice:)

Reply:
I would like to have it part of another PR if possible. Otherwise,  it would be more like enhancement + fix pep8 violation rather than fix pep8 violation

Reply:
Sure:)

Reply:
Thanks :)

Reply:
@neesingh-rh If it's good to go, do approve the changes.